src/main/java/frc/team2412/robot/subsystem/ShooterVisionSubsystem.java

```diff
@@ -4,15 +4,16 @@
 import edu.wpi.first.networktables.NetworkTable;
 import edu.wpi.first.networktables.NetworkTableInstance;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import io.github.oblarg.oblog.Loggable;
+import io.github.oblarg.oblog.annotations.Log;
 
-public class ShooterVisionSubsystem extends SubsystemBase {
-
+public class ShooterVisionSubsystem extends SubsystemBase implements Loggable {
     public static class ShooterVisionConstants {
-
-        public static double LIMELIGHT_ANGLE_OFFSET = 56.3;
-        public static double LIMELIGHT_HEIGHT_OFFSET = 39;
-        public static double RIM_HEIGHT = 104; // 8ft8in
-        public static double HEIGHT_TO_RIM = RIM_HEIGHT - LIMELIGHT_HEIGHT_OFFSET;
+        public static final double LIMELIGHT_ANGLE_OFFSET = 10.95;
```

Comment:
The angle changed again?

Reply:
We changed the reference crosshair of the limelight so that 0 degrees from the limelight means 360 inches from the center of the hub. We could enter an equation so compiler calculates that for us.